test/draw/RasterCanvasTest.ooc

```diff
@@ -23,7 +23,7 @@ RasterCanvasTest: class extends Fixture {
 			halfSize := image size / 2
 			image drawLine(FloatPoint2D new(-halfSize x, -halfSize y), FloatPoint2D new(halfSize x, halfSize y), Pen new(ColorRgb new(0, 255, 0)))
 			image drawLine(FloatPoint2D new(halfSize x, -halfSize y), FloatPoint2D new(-halfSize x, halfSize y), Pen new(ColorRgb new(0, 255, 0)))
-			expect(image distance(correctImage), is equal to(0.0f))
+			expect(image distance(correctImage), is less than(0.01f))
```

Comment:
This syntax is quite messy but we use it to print both values on assertion failure.
In this case, precision comes from 8-bit rounding but inherited by the float so it has to be higher than usual.